condacolab.py

```diff
@@ -89,11 +89,38 @@ def install_from_url(
         stderr=STDOUT,
         text=True,
     )
+
+    # installing google.colab packages, matplortlib-base and psutil.
+
+    conda_exe = "mamba" if os.path.isfile(f"{prefix}/bin/mamba") else "conda"
+
+    psutil_task = run(
+        [f"{prefix}/bin/{conda_exe}", "install", "-yq", "matplotlib-base", "psutil", "google-colab"],
+        check=False,
+        stdout=PIPE,
+        stderr=STDOUT,
+        text=True,
+    )
+
+    colab_task = run(
+        [f"{prefix}/bin/python", "-m", "pip", "-q", "install", "https://github.com/googlecolab/colabtools/archive/refs/heads/main.zip", "condacolab"],
```

Comment:
```suggestion
        [f"{prefix}/bin/python", "-m", "pip", "-q", "install", "-U", "https://github.com/googlecolab/colabtools/archive/refs/heads/main.zip", "condacolab"],
```

In case the installers come with colabtools or something, we might want to force the update.